plugins/feeds/public/botvrij_url.py

```diff
@@ -1,38 +1,39 @@
 import logging
 from datetime import timedelta, datetime
-from core.errors import ObservableValidationError
-from core.feed import Feed
-from core.observables import Url
+from core.schemas import observable
+from core.schemas import task
+from core import taskmanager
 
 
-class BotvrijUrl(Feed):
-    default_values = {
+class BotvrijUrl(task.FeedTask):
+    SOURCE = "https://www.botvrij.eu/data/ioclist.url"
+
+    _defaults = {
         "frequency": timedelta(hours=12),
         "name": "BotvrijUrl",
-        "source": "https://www.botvrij.eu/data/ioclist.url",
-        "description": "Detect possible outbound malicious activity.",
+        "description": "Botvrij.eu is a project of the Dutch National Cyber Security Centre (NCSC-NL) and SIDN Labs, the R&D team of SIDN, the registry for the .nl domain.",
     }
 
-    def update(self):
-        resp = self._make_request(sort=False)
-        lines = resp.content.decode("utf-8").split("\n")[6:-1]
-
-        for line in lines:
-            self.analyze(line.strip())
+    def run(self):
+        response = self._make_request(self.SOURCE, verify=True)
```

Comment:
Should we make `verify=True` by default in `_make_request`? It seems like we're always passing it.

Reply:
Yep